lib/data/locationConstraintParser.js

```diff
@@ -0,0 +1,41 @@
+import file from './file/backend';
+import inMemory from './in_memory/backend';
+import Sproxy from 'sproxydclient';
+
+import config from '../Config';
+
+const clients = {};
+
+export default function parseLC() {
+    if (config.locationConstraints) {
+        Object.keys(config.locationConstraints).forEach(location => {
+            const locationObj = config.locationConstraints[location];
+            if (locationObj.type === 'mem') {
+                clients[location] = inMemory;
+            }
+            if (locationObj.type === 'file') {
+                clients[location] = file;
+            }
+            if (locationObj.type === 'scality'
+            && Object.keys(locationObj.details.connector).
```

Comment:
I think we can just check whether locationObj.details.connector.sproxyd is defined